### ISSUE TITLE
Tests failing

### ISSUE THREAD

**Issue by GrahamCampbell** [php-ds/polyfill#58]
```
There were 3 errors:

1) Ds\Tests\PairTest::testBadPropertyAccess

Undefined property: Ds\Pair::$nope

/home/travis/build/php-ds/polyfill/vendor/php-ds/tests/tests/Pair/__get.php:18

2) Ds\Tests\PairTest::testPropertyUnsetKey

Undefined property: Ds\Pair::$key

/home/travis/build/php-ds/polyfill/vendor/php-ds/tests/tests/Pair/_unset.php:10

3) Ds\Tests\PairTest::testPropertyUnsetValue

Undefined property: Ds\Pair::$value

/home/travis/build/php-ds/polyfill/vendor/php-ds/tests/tests/Pair/_unset.php:17
```

**Comment by simPod** [comment]
I "fixed" 1 in https://github.com/php-ds/tests/pull/12

I believe 2 and 3 behaviour is invalid.

```php
    public function testPropertyUnsetValue()
    {
        $pair = $this->getPair('a', 1);
        unset($pair->value);
        $this->assertNull($pair->value);
    }
```

When property is unset, it is not always `null` https://3v4l.org/WKiMJ

**Comment by rtheunissen** [comment]
This should now be fixed as part of 1.3.0

**Comment by GrahamCampbell** [comment]
🚀
